crates/turbopack-tests/js/jest-entry.ts

```diff
@@ -0,0 +1,23 @@
+declare const __turbopack_external_require__: (id: string) => any;
+const jest = __turbopack_external_require__("jest-circus");
+
+const uncaughtExceptions: string[] = [];
+const unhandledRejections: string[] = [];
+
+process.on("uncaughtException", (e) => {
+  uncaughtExceptions.push(String(e));
+});
+
+process.on("unhandledRejection", (e) => {
+  unhandledRejections.push(String(e));
+});
```

Comment:
Can you verify that these actually capture in jest? I think it actually unregisters both when running tests to install their own listeners, because Jest loves to screw everything up.

Reply:
Has this been verified?

Reply:
yeah, via an unhandled Promise.reject in test code.